[2025-04-03T14:44:53.748+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-03T14:44:53.774+0000] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: tmdb_pipeline.process_kafka_data scheduled__2025-03-31T12:00:00+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2025-04-03T14:44:53.793+0000] {local_task_job_runner.py:163} INFO - Task is not able to be run
